test/tests/restart/restart_subapp_not_master/two_step_solve_sub_restart.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  file = two_step_solve_master_cp/0002_mesh.cpr
+[]
+
+[Problem]
+  restart_file_base = two_step_solve_master_cp/LATEST
+  allow_sub_app_restart = true
```

Comment:
I really think it's enough to just specify a restart file in the sub-app.  It's exactly explicit in what they want to do.

But - I understand you want to be extra careful here.  Out of the choices I would vote for `force_restart`.  I just don't think users will understand _why_ they have to do anything "special" beyond specifying a restart file (we know why because we know the internal code).

I guess what I'm trying to say is: don't think about how it's implemented on the inside... think about how a user will see it.